tf2onnx/tf_loader.py

```diff
@@ -96,7 +96,19 @@ def inputs_without_resource(sess, input_names):
     return input_names
 
 
-def from_function(func, input_names, output_names):
+def from_function(func, input_names, output_names, large_model):
```

Comment:
maybe use large_model=None since one might call this from some random python code

Reply:
other than that super lgtm.